### ISSUE TITLE
Unknown: PID Not Found

### ISSUE THREAD

**Issue by Sircam19** [GetiPlayerAutomator/get-iplayer-automator#474]
Anyone having an issue with GiA that corresponds. This message is coming up with both radio downloads and TV.

**Comment by dinkypumpkin** [comment]
If you can't even be bothered to provide an example, how can you expect a meaningful response?

**Comment by Sircam19** [comment]
Your absolutely right, my apologies.  Sorry for wasting your time, you have done excellent work and I along with many other appreciate it, greatly. 

This is my situation: All of the queued downloads have reverted to the following but by way of example: 
PID       Status             Show Name - Episode Name
1319     Not Available  Unknown: PID Not Found

No new shows (e.g The Andrew Marr Show, broadcast today - Sunday) populate the download queue. 

Interestingly, and this is what I noticed after my post is that using Safari AND on all of our machines at home cannot access any BBC website, but Firefox and Chrome will. 

I changed the default browser from Safari to Chrome but still get the following  when I hit Show log... but nothing else happens....afterwards. 

Get iPlayer Automator 1.8.5 Initialized.

AppController: Updating Program Index Feeds...

AppController: Index was Up-To-Date.

Does anything immediately come to mind as far as a fix...cheers. 

S
